snap/info_test.go

```diff
@@ -128,6 +128,10 @@ func (s *infoSuite) TestReadInfo(c *C) {
 }
 
 func makeTestSnap(c *C, yaml string) string {
```

Comment:
Ah, `MakeTestSnapWithFiles` is new, I wrote this before it existing and then merged it in without noticing. I'll begin using it instead.